x-pack/plugins/enterprise_search/common/connectors/native_connectors.ts

```diff
@@ -828,4 +828,94 @@ export const NATIVE_CONNECTOR_DEFINITIONS: Record<string, NativeConnector | unde
     }),
     serviceType: 'postgresql',
   },
+  sharepoint_online: {
+    configuration: {
+      tenant_id: {
+        default_value: null,
+        depends_on: [],
+        display: DisplayType.TEXTBOX,
+        label: 'Tenant Id',
```

Comment:
All of the labels and tooltips in here need to be translated using `i18n` functions. See the examples in the other connectors on how to do that. 

Also, we've generally used `Sentence case` instead of `Title Case` for these values. So: `Tenant Name` instead of `Tenant Name`, and so on for all the others. 


`Tenant ID` or `Tenant id` are both preferable to`Tenant Id`.

Reply:
Ah I overlooked it, thanks!